### PR TITLE
Add post on signing disruption

### PR DESCRIPTION
**Pull request by the-maldridge**

### CODE REVIEW

File: _posts/2021-07-17-signing-disruption.markdown [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Repository Signing Disruption
+layout: post
+---
+
+We are aware of an ongoing infrastructure issue that is affecting
+repository syncronization and package signing.  We are working to
```
Comment (fosslinux) [inline _posts/2021-07-17-signing-disruption.markdown:7]: syncronization -> synchronization

File: _posts/2021-07-17-signing-disruption.markdown [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+resolve these issues, but first must resolve another unrelated
```
Comment (fosslinux) [inline _posts/2021-07-17-signing-disruption.markdown:8]: Perhaps make it clear whether it is one issue or not, in first sentence you say "issue" but here it is "issues"

File: _posts/2021-07-17-signing-disruption.markdown [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: Repository Signing Disruption
+layout: post
+---
+
+We are aware of an ongoing infrastructure issue that is affecting
```
Comment (ahesford) [inline _posts/2021-07-17-signing-disruption.markdown:6]: that is affecting -> affecting

File: _posts/2021-07-17-signing-disruption.markdown [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+repository synchronization and package signing.  We are working to
+resolve this disruption, but first must resolve another unrelated
+infrastructure issue.  We have no short term ETA for service
+restoration, however our worst case resolution timeline is Friday the
+23rd of July.
+
+Your patience is appreciated while we perform this work as fast as we
+are able to.
```
Comment (ahesford) [inline _posts/2021-07-17-signing-disruption.markdown:14]: as fast as we are able to -> as fast as possible

**Pull request merged**